python/graphstorm/eval/eval_func.py

```diff
@@ -192,62 +194,112 @@ def eval_roc_auc(logits,labels):
     predicted_labels=logits
     predicted_labels=predicted_labels.detach().cpu().numpy()
     labels=labels.detach().cpu().numpy()
+
+    # check if the two inputs have the same number of rows.
+    try:
+        assert predicted_labels.shape[0] == labels.shape[0]
+    except AssertionError:
+        logging.error('Predictions and labes should have ' + \
+                f'the same number of records, but got, {predicted_labels.shape[0]} and ' + \
+                f'{labels.shape[0]}.')
+        raise
+
+    # check if the predictions >=2D.
+    try:
+        assert len(predicted_labels.shape) > 1
```

Comment:
If you assume predicted_labels is a 2D tensor, then `assert len(predicted-lables.shape) == 1`